src/Uno.Core/Options/Option.cs

```diff
@@ -29,7 +29,7 @@ public abstract class Option
 		/// <summary>
 		/// Creates an option for a given value.
 		/// </summary>
-		public static Some<T> Some<T>(T value) => new Some<T>(value);
+		public static Option<T> Some<T>(T value) => new Some<T>(value);
```

Comment:
Alleluia

Reply:
That's a binary breaking change... why is it needed ?

Reply:
It has been delayed in the past because of this reason. Was waiting for the right moment to push it. We can put it back on backburner if you want.

Reply:
Has been moved to pr #11